src/wp-admin/upload.php

```diff
@@ -141,7 +141,13 @@
 	wp_enqueue_script( 'media-grid' );
 	wp_enqueue_script( 'media' );
 
-	remove_action( 'admin_head', 'wp_admin_canonical_url' );
+	// Remove the error parameter added by deprecation of wp-admin/media.php.
+	add_filter(
+		'removable_query_args',
+		function( $query_args ) {
```

Comment:
@peterwilsoncc Should we just remove this parameter from the callback signature since it isn't used in the callback?